nipoppy/workflow/bids_conv/run_dcm2bids.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python
```

Comment:
Could you change file permissions to make it executable (`chmod u+x run_dbm2bids.py`)? I don't think that's the case right now